lib/collections.js

```diff
@@ -43,12 +45,19 @@ var uniqueWords = [];
 var longAndShort = {};
 
 //jscs:disable requireDotNotation
-longAndShort['longest'] = '';
-longAndShort['shortest'] = paragraph;
-
-/* your code starts here */
-
-/* your code ends here */
+//longAndShort['longest'] = '';
+//longAndShort['shortest'] = paragraph;
+
+// 1. iterate through the array
+// 2. for each element, use the length method to determine its length
+// 3. store that key/value pair in the longAndShort associative array
+// 4. Read through the hash, selecting the smallest and largest values
+
+for (var i = 0; i < uniqueWords; i++) {
+  longAndShort[i] = uniqueWords[i];
+  longAndShort[uniqueWords[i]] = longAndShort[i].length;
+}
+console.log(longAndShort);
```

Comment:
At the moment, this is what your code says.
1. Iterate through every word in the list `uniqueWords`.
2. For each word, add an entry to the `longAndShort` dictionary, where the key is our current index in the `uniqueWords` array and the value is the current word.
3. For each word, add an entry to the `longAndShort` dictionary, where the key is the current word and the value is that word's length. 

While this will, indirectly, build a dictionary of words and their lengths, it does not accomplish the objective of finding the longest and shortest words.